framework/src/problems/FEProblemBase.C

```diff
@@ -5060,7 +5060,7 @@ FEProblemBase::computeResidualTags(const std::set<TagID> & tags)
       }
       catch (libMesh::LogicError & e)
       {
-        throw MooseException("We caught a libMesh error");
+        throw MooseException("We caught a libMesh error in FeProblemBase");
```

Comment:
If we're going to do the `min_parallel` change might as well changes this to `FEProblemBase`

Reply:
?

Reply:
Oh, caps, gotcha